openapi/endpoints/login/models/challenge-request.yaml

```diff
@@ -2,18 +2,36 @@ type: object
 required:
   - walletAddress
   - chainId
+  - clientApplicationId
+  - scopes
 properties:
   walletAddress:
     description: The address of the signing wallet or the smart contract that implements ERC-1271
     type: string
-    example: "0xA3058369d6A481B1ff08F62B352409c3D709De9b"
+    example: '0xA3058369d6A481B1ff08F62B352409c3D709De9b'
   chainId:
     description: Chain ID (see https://chainlist.org/)
     type: number
     example: 1
+  clientApplicationId:
+    description: Client application ID assigned by IMMERSVE
+    type: string
+    example: deeec538c6e59f8e5c746c05f766fcc0
+  scopes:
+    description: An array of authorization scopes requested by the client application
+    type: array
+    items:
+      type: string
+      enum:
+        - full-access
+        - full-access:partner
+        - cardholder-partner
+        - update-kyc-profile
```

Comment:
we need a description of these as well, in the Authentication guide

Reply:
@rb455 Do you think it would be good to have a simple endpoint to list scopes, along with their descriptions? Then we wouldn't need to maintain the docs for it

Reply:
I can update the authentication guide for now though

Reply:
that's an interesting approach, yeah, we can do it that way at some point

Reply:
added description